src/mudlet.pro

```diff
@@ -777,18 +777,19 @@ HEADERS += \
     ../3rdparty/discord/rpc/include/discord_register.h \
     ../3rdparty/discord/rpc/include/discord_rpc.h
 
-macx {
-    OBJECTIVE_SOURCES += AnnouncerMac.mm
-}
-
-win32 {
-    SOURCES += AnnouncerWindows.cpp \
-        uiawrapper.cpp
+macx|win32 {
+    macx {
+        OBJECTIVE_SOURCES += AnnouncerMac.mm
```

Comment:
This file is only needed on macOS, not on Windows as well.

Reply:
Which is what the ***inner*** `macx` does - for clarity I did put the Windows one in it's own `win32` group below rather than just using `} else {` - but the reasoning is the same as the existing structure in the `CMakeList.txt` file - do this for MacOs, do that for Windows otherwise do something else for *all other cases*...

Do you, like me, need another cup of :coffee: ? :slightly_smiling_face: